Payments/UserInterface/01a_ListAuthorizations.md

```diff
@@ -0,0 +1,142 @@
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+    Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.
+
+The list displays all authorization transactions. Depending on the payment service provider, the displayed columns vary. All fields are read-only. The following standard columns are available.
+<!----Hallo Stefan, hier sollten alle Standard-Felder mit Erklärungen stehen, ist die Liste vollständig?--->
+- *Charges intended use*   
+   Purpose of the fee   
+- *Original currency*   
+   Currency in which the product(s) were paid   
+- *Original currency amount*   
+   Invoice amount in the original currency   
+
+
+
+## Authorizations &ndash; Attributes
+
+*Payments > Payments and Captures > Tab LIST > Select an authorization ticket > Tab Attributes*   
+
+
+![LIST &ndash; Attributes](../../Assets/Screenshots/Payments/Authorizations/AuthorizationAttributes.png "[LIST &ndash; Attributes]")
+
+In the left margin column, all available attribute groups are displayed. Click a group to display the attributes that are assigned to this group on the right side of the *Attributes* tab. If the authorization transaction contains attributes that are not assigned, the *Unassigned group* attribute group is automatically displayed in the left margin column.
+
+- ![Folders](../../Assets/Icons/Folders01.png "[Folders]") (Folders)  
+    Attribute group that contains attribute sub-groups. Click the attribute group or the arrow left to the attribute group to expand the group and to display the attribute sub-groups. The following sub-groups are available:
+    - *[Payment service provider] Incoming Payment Base Data*   
+      This folder contains attributes that are specific for the payment service provider who has sent this authorization ticket.
+    - *Incoming Payment Base Data*   
+      This folder contains general fields that Actindo needs to query payments data. These data support standard functions of Actindo. 
+
+- ![Folder](../../Assets/Icons/Folder01.png "[Folder]") (Folder)  
+    Attribute group. Click the attribute group to display all attributes that are assigned to the selected attribute group on the right side of the *Attributes* tab.
+
+The right side of the *Attributes* tab displays all attributes that are assigned to the selected group in the left margin column. Depending on the selected attribute group, the displayed fields may vary. The attributes are imported by the payment service provider driver. All fields are read-only.
+
+
+## Authorizations &ndash; Dependencies
+*Payments > Menu entry > Tab LIST > Select an authorization ticket > Tab Dependencies*
+
+![Dependencies](../../Assets/Screenshots/Payments/Authorizations/AuthorizationDependencies.png "[Dependencies]")   
+
+
```

Comment:
Könnte es geben? Bitte mit Stefan klären. Das habe ich ursprünglich auch in Fulfillment gedacht...
Wenn nötig, kannst du die Beschreibung zu Standard UI von Fulfillment oder Omni-Channels kopieren.

Reply:
schon mit Stefan geklärt